### PR TITLE
Outfalls tidal curve hours as float number (not HH.MM)

### PR DESCRIPTION
**Pull request by FLO-2DJJ**
- First column of tidal curve table (hours) accepts real numbers.
- Items in Tidal curve table is loaded in ascending order by first column (hours).
- Cells of table only accepts digits, period and minus sign.

### CODE REVIEW

File: flo2d/ui/storm_drain_outfall_tidal_curve.ui
Comment (FLO-2DKaren) [inline flo2d/ui/storm_drain_outfall_tidal_curve.ui:1]: @FLO-2DJJ 
Hi JJ,  I'm learning how the Review tools work.  I'm trying to add a comment.  to this UI file.
Tide data can be very long and typically comes in 6 min intervals.  We need a way to import a file. 
image

I attached some test data that I found on NOAA Tide website.
[TideData.txt](https://github.com/FLO-2DSoftware/qgis-flo-2d-plugin/files/11179134/TideData.txt)

Comment (FLO-2DKaren) [reply flo2d/ui/storm_drain_outfall_tidal_curve.ui:1]: I also think you can do add the Paste Option since we are not needing this right away.
image

**Review by FLO-2DKaren — CHANGES_REQUESTED** [review]
See comments in UI file.

**Comment by FLO-2DJJ** [comment]
- I will create a new issue with @FLO-2DKaren observations, namely, an import file button and paste button for the Tidal Curve dialog.
- Meanwhile I merge this pull request into main.

**Pull request merged**